src/common/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { ReactNode, useState } from 'react'
+import ReactDatePicker from 'react-datepicker'
+import cn from 'classnames'
+// Components
+import { Input } from '../Inputs'
+// Assets
+import CalendarIcon from 'common/assets/icons/ico-calendar.svg'
+import ChevronIcon from 'common/assets/icons/ico-chevron.svg'
+
+import './DatePicker.module.css'
```

Comment:
NB: it works as expected, so not a problem. CSS modules a bit different from not modules: `import style from 'x'` this syntax for a css module, and then you can use class names using it. While `import 'x'` is not a module, more likely it is regular css injected to the page.

So no need for `.module.` suffix in the file name if it is not a module.